### PR TITLE
Board class david

### PR DESCRIPTION
**Pull request by dkassin**
We have 13 tests that are passing, some tests have been x'd out. 
I changed the class variable that referred to a hash object to class_hash bc it was in conflict with our class method. 
This allowed for all tests to pass and place method to change the class_hash variable as expected.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  def place(ship_type, coordinates)
+    @cell_hash
+    coordinates.each do |coordinate|
+      cell = Cell.new(coordinate)
+      cell.place_ship(ship_type)
+      @cell_hash[coordinate] = cell
+    end
+    @cell_hash
+  end
+  require "pry"; binding.pry
```
Comment (kbhoffmann) [inline lib/board.rb]: You just forgot your pry in here on line 87.

File: spec/board_spec.rb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    cruiser = Ship.new("Cruiser", 3)
+    submarine = Ship.new("Submarine", 2)
+
+    expect(board.valid_placement?(cruiser, ["A1", "B2", "C3"])).to eq(false)
+    expect(board.valid_placement?(submarine, ["C2", "D3"])).to eq(false)
+    expect(board.valid_placement?(submarine, ["A1", "A2"])).to eq(true)
+    expect(board.valid_placement?(cruiser, ["B1", "C1", "D1"])).to eq(true)
+  end
+
+  it 'can place ships in its cells ' do
```
Comment (kbhoffmann) [inline spec/board_spec.rb:93]: After class would you mind explaining to me how this method works?  I haven't had a chance to explore this method prior so just wanted to know what its doing exactly.

**Review by kbhoffmann — COMMENTED** [review]
Looks nice and clean!  Thank you for finishing up the 'valid_placement?' method and taking on the 'place' method.

I just left a comment about removing the pry from line 87 of the class file.

File: lib/board.rb (hunk 1 above)
Comment (dkassin) [reply lib/board.rb]: whoops I will remove this, great catch.

File: spec/board_spec.rb (hunk 2 above)
Comment (dkassin) [reply spec/board_spec.rb:93]: Yes no problem.

**Pull request closed without merging**